go/libkb/constants.go

```diff
@@ -722,3 +721,5 @@ const (
 )
 
 const ProfileProofSuggestions = true
+
+const ShutdownChanDefaultSize = 10
```

Comment:
But more specifically, why `10`? Also you replaced some numbers smaller than 10 with this.

Reply:
10 was arbitrary but I'm going to replace the recent changes to address the flakes with the new pattern we discussed anyway.